Proxies/Numbers.cs

```diff
@@ -29,7 +29,7 @@ public abstract class IntegerProxy {
         }
 
         [JSIsPure]
-        [JSReplacement("($this === $rhs)")]
+        [JSReplacement("(($this).valueOf() === ($rhs).valueOf())")]
```

Comment:
Oh, though one caveat is that ```x >>> 0 === y >>> 0``` will be wrong if one of the operands is signed, since it'll force it to unsigned. Hm.